addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -442,6 +442,23 @@ def test_message_process_email_author_exclude_alias(self):
         self.assertFalse(record.message_ids[0].author_id)
         self.assertEqual(record.message_ids[0].email_from, from_1.email_formatted)
 
+    def test_message_parse_invalid_encoding_in_attachment(self):
```

Comment:
I would put it after 'test_message_process_file_encoding' (in a file section about corner cases / bugs during message process), could be put after and named like 'test_message_process_file_encoding_ascii' maybe ? Just to order a bit tests and try to have them a bit grouped :)